package.json

```diff
@@ -66,12 +66,13 @@
     "lint:fix": "lerna run lint:fix",
     "publish": "node scripts/publish-circleci.js",
     "test": "lerna exec --concurrency 1 --stream --bail=false -- npm run test --if-present",
-    "test:unit": "lerna exec --concurrency 1 --stream --bail=false -- npm run test:unit --if-present",
+    "test:unit": "lerna run test:unit",
     "test:ui": "lerna exec --concurrency 1 --stream --bail=false -- npm run test:ui --if-present",
     "test:unit-and-system-tests": "lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test:unit --if-present && npm run coverage:system-tests",
+    "test:download-vscode": "node scripts/download-vscode",
     "test:integration": "lerna run test:integration --concurrency 1 --stream -- --if-present",
-    "test:integration-tests-without-system-tests": "node scripts/download-vscode && lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test:integration --if-present && npm run test:vscode-integration --if-present",
-    "test:vscode-integration": "node scripts/download-vscode && lerna exec --concurrency 1 --stream --bail=false -- npm run test:vscode-integration --if-present",
+    "test:integration-tests-without-system-tests": "npm run test:download-vscode && lerna run test:integration && lerna run test:vscode-integration --concurrency 1",
```

Comment:
the `test:vscode-integration` is the only tests that get grumpy if you try to paralyze due to launching vscode.  Hence it's the only one that keeps the concurrency flag.